packages/cron-job/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "draw/cron-job",
+    "description": "This component allows to manage cron jobs.",
+    "license": "MIT",
+    "type": "library",
+    "keywords": ["draw", "cronjob", "cron", "cron job"],
+    "authors": [
+        {
+            "name": "Martin Poirier Theoret",
+            "email": "mpoiriert@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=8.1"
```

Comment:
don't hesitate to use the dragonmantank/cron-expression or something else for the cron schedule portion.

I never check which is the better package but it's the one that was use in pnp.